### ISSUE TITLE
tracepath documentation still references a tracepath6 binary

### ISSUE THREAD

**Issue by nmeyerhans** [iputils/iputils#98]
Forwarded from https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=869202

The [tracepath documentation](https://github.com/iputils/iputils/blob/master/doc/tracepath.xml) still contains multiple references to a tracepath6 binary, which we no longer build or install.

From the original report:

```
The tracepath(8) man page mentions tracepath6:

NAME
   tracepath, tracepath6 - traces path to a network host discovering MTU
              ^^^^^^^^^^
   along this path
[...]
   tracepath6 is good replacement for traceroute6 and classic example of
   ^^^^^^^^^^
[...]
   -l     Sets  the initial packet length to pktlen instead of 65535 for
          tracepath or 128000 for tracepath6.
                                  ^^^^^^^^^^
[...]
OUTPUT
       root@mops:~ # tracepath6 3ffe:2400:0:109::2
                     ^^^^^^^^^^

However, tracepath6 no longer exists. The Debian changelog file says:

  * Merge IPv6 support into tracepath, drop tracepath6 binary.

```

**Comment by okias** [comment]
at least for ping, distribution sometimes link ping as ping6. I think it's not necessary for tracepath, so feel free to push fix :) since we have XML now, it should be piece of cake :)

**Comment by nmeyerhans** [comment]
Yes, we ship a ping6 link in Debian currently, but we don't do that for tracepath6.

I'll send a PR to update the XML...

**Comment by kerolasa** [comment]
The tracepath6 fix was done in commit 52370fa17af1f007af1b589e951b5cfc628a6df6
